apiv2/support/enums.py

```diff
@@ -62,35 +62,35 @@ class sample_type_enum(enum.Enum):
 
 
 @strawberry.enum
-class tiltseries_camera_acquire_mode_enum(enum.Enum):
+class tiltseries_camera_acquire_mode_enum(enum.StrEnum):
     counting = "counting"
     superresolution = "superresolution"
     linear = "linear"
     cds = "cds"
 
 
 @strawberry.enum
-class tiltseries_microscope_manufacturer_enum(enum.Enum):
+class tiltseries_microscope_manufacturer_enum(enum.StrEnum):
```

Comment:
Should this be an enum at the db level?

Reply:
Yep! these enums are mapped to our SQLAlchemy models, like here for example:
https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/main/apiv2/database/models/tiltseries.py#L75-L77